src/main/webapp/app/course/learning-paths/learning-path-management/learning-path-management.component.html

```diff
@@ -33,6 +33,25 @@ <h2 jhiTranslate="artemisApp.learningPath.manageLearningPaths.title"></h2>
             <jhi-learning-path-health-status-warning [status]="HealthStatus.NO_RELATIONS" (onButtonClicked)="routeToCompetencyManagement()" />
         }
         @if (!health.status?.includes(HealthStatus.DISABLED)) {
+            <div class="panel-wrapper">
+                <h3 class="panel-header" jhiTranslate="artemisApp.learningPath.manageLearningPaths.configuration.title"></h3>
+                <div class="vertLine"></div>
+                <div class="panel-body">
+                    <input
+                        type="checkbox"
+                        id="includeAllGradedExercises"
+                        (change)="updateLearningPathsConfiguration()"
+                        [(ngModel)]="learningPathsConfiguration.includeAllGradedExercises"
```

Comment:
Could `learningPathsConfiguration` be null/undefined here? When opening my course page on TS5, I received thousands of errors in the console